e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -121,13 +123,13 @@ Scenario('Gatekeeper drafts standard directions', async (I, caseViewPage, draftS
   I.seeEventSubmissionConfirmation(config.administrationActions.draftStandardDirections);
 
   caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
-  I.seeInTab(['Standard directions order', 'File'], 'draft-standard-directions-order.pdf');
-  I.seeInTab(['Standard directions order', 'Date of issue'], dateFormat(today, 'd mmmm yyyy'));
+  I.seeInTab(['Gatekeeping order', 'File'], 'draft-standard-directions-order.pdf');
+  I.seeInTab(['Gatekeeping order', 'Date of issue'], dateFormat(today, 'd mmmm yyyy'));
   I.seeInTab(['Directions 1', 'Title'], 'Request permission for expert evidence');
   I.seeInTab(['Directions 1', 'Description'], 'Your request must be in line with Family Procedure Rules part 25 and Practice Direction 25C. Give other parties a list of names of suitable experts.');
   I.seeInTab(['Directions 1', 'For'], 'All parties');
   I.seeInTab(['Directions 1', 'Due date and time'], '1 Jan 2050, 12:00:00 PM');
-});
+}).retry(2); // Retry on error, this should help with the data altered outside of transaction issue
```

Comment:
See https://codecept.io/basics/#retry-scenario
You cannot have conditional retries on a scenario retry otherwise I would try and check if the error
```
data altered outside of transaction issue
```
was reported and only retry then

Reply:
I don't know what we have done but reties doesn't work with our project. I have with both variations described in the page above and it doesn't retry.

I have also tried generating a new codecept test project following https://codecept.io/quickstart/ with the following test
```js
Feature('foo');

Scenario('bar', (I) => {
  I.say('hi');
  throw new Error('boom');
}).retry(2);
```
and it worked fine
```
❯ npx codeceptjs run --steps
CodeceptJS v2.6.10
Using test root "/Users/lewisb@kainos.com/work/test"

foo --
  bar
   hi
   hi
   hi
  ✖ bar in 244ms
  S bar

-- FAILURES:

  1) foo
       bar:
     boom
```

Reply:
See #1594